build.gradle

```diff
@@ -85,6 +86,9 @@ apply from: "gradle/gradle-plugin-portal.gradle"
 
 tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile) {
     kotlinOptions {
-        allWarningsAsErrors = true
+        // Disabling until there's some workaround for the warning
+        // about AGP's builder.jar containing its own bundled Kotlin
+        // runtime library.
+        allWarningsAsErrors = false
```

Comment:
The warning specifically is that `com.android.tools.build:builder:3.3.0-rc03` bundles the Kotlin runtime library.  It's not a dependency, and so can't be excluded or overridden like we normally would do.  I haven't figured out a good way to silence this specific warning.  If there's a better way, I'm all for it!

Reply:
It is `false` by default. Thanks for adding this comment. Hopefully we can specifically disable individual warnings in the future. We can just be careful not to add more warnings.